$OpenBSD: patch-src_mod_fastcgi_c,v 1.12 2013/03/28 16:22:32 brad Exp $

- Fix mix up of "mode" => "authorizer" in other FastCGI configs.
- Support multi line logging.

--- src/mod_fastcgi.c.orig	Sat Nov 17 12:32:14 2012
+++ src/mod_fastcgi.c	Wed Mar 27 09:16:41 2013
@@ -1275,6 +1275,7 @@ SETDEFAULTS_FUNC(mod_fastcgi_set_defaults) {
 					}
 
 					host = fastcgi_host_init();
+					buffer_reset(fcgi_mode);
 
 					buffer_copy_string_buffer(host->id, da_host->key);
 
@@ -2673,8 +2674,8 @@ static int fcgi_demux_response(server *srv, handler_ct
 		case FCGI_STDERR:
 			if (packet.len == 0) break;
 
-			log_error_write(srv, __FILE__, __LINE__, "sb",
-					"FastCGI-stderr:", packet.b);
+			log_error_write_multiline_buffer(srv, __FILE__, __LINE__, packet.b, "s",
+					"FastCGI-stderr:");
 
 			break;
 		case FCGI_END_REQUEST:
